README.md

```diff
@@ -400,7 +400,7 @@ do.MustShutdownNamed(injector, "configuration")
 
 ### Service override
 
-By default, providing a service twice will panic. Service can be replaced at runtime using `do.Replace****` helpers.
+By default, providing a service twice will panic. Service can be replaced at runtime using `do.Override` helper.
```

Comment:
Currently, functions named `do.Replace****` are not found and this name seems to another candidate of `do.Override` .

https://github.com/samber/do/pull/6#issuecomment-1150584002